### DISCUSSION TITLE
(INVALID) Keta vs MacZip: 7-Zip Slowest Compression PDFs Test / Benchmark

### DISCUSSION THREAD

**Discussion by NamelessGO** [aonez/Keka#1524] [Show and tell]
This post is invalid!

**Comment by aonez** [comment]
Usually this kind of test is done with public files so it can be trusted and tested. I did a quick test with this file:
https://github.com/mozilla/DeepSpeech/releases/download/v0.10.0-alpha.3/deepspeech-0.10.0-alpha.3.tgz

Using a MacBook Pro M2 32GB. Original size of the package once extracted: 120.3MB

- Keka: 17.4MB (85.5%) in 12.8 seconds
- MacZip: 20.3MB (83.1%) in 17.4 seconds

Very curious to know what you've compressed there.

<img width="669" alt="Screenshot 2024-10-02 at 16 47 01" src="https://github.com/user-attachments/assets/22458b2d-8f04-4bd0-9acb-07d80510345c">

<img width="565" alt="Screenshot 2024-10-02 at 16 46 33" src="https://github.com/user-attachments/assets/779d425c-7198-4c96-bea9-ee19c3d5f097">

**Reply by NamelessGO** [reply]
Should have used sample files for sure, originally was going to try a bunch of file archive apps to see decrease as much as possible. But because the size gap were considerable large, decided to make a post to share the findings

With your test could see how flawed my test was

Also Tried using PeaZip with the slowest setting "Ultra", but the result is **way closer** to Keka than MacZip



> Very curious to know what you've compressed there.

The file contains a lot of folders inside folders, watermarks, and images, but this don't say a lot tbh, can't be public either

**Comment by gingerbeardman** [comment]
After setting Keka options, how are you compressing? By dragging folder to Keka window?

I ask because your screenshot is the Keka preferences that are used only when dragging onto the window. 

There are different default preferences used for the "Compress with Keka" menu item, dragging onto the app icon, etc.

**Reply by NamelessGO** [reply]
Dragging folder to Keka window, learned later that you can drag and drop to keka directly in the docker

**Comment by NamelessGO** [comment]
Sorry for the post, this test is invalid

These files are copyrighted, so can't share publically

Tried to see a way to anonymize files to send, but the max on Github is 25mb while the minimum folder here is 250mb+

The best course of action would prob be to delete the post

Thanks for the work on Keka, got the recommendation while searching for file archive on reddit

**Reply by aonez** [reply]
No problem @NameLessGO, it was useful to do some tests.
